packages/services/src/services/SearchTypeahead/SearchTypeahead.js

```diff
@@ -39,11 +40,10 @@ class SearchTypeaheadAPI {
    * }
    */
   static async getResults(query) {
-    const lc = 'en'; // TODO: create utility for fetching lc
-    const cc = 'us'; // TODO: create utility for fetching cc
-    const url = `${_endpoint}?lang=${lc}&cc=${cc}&query=${encodeURIComponent(
-      query
-    )}`;
+    const lang = LocaleAPI.getLang();
+    const url = `${_endpoint}?lang=${lang.lc}&cc=${
+      lang.cc
+    }&query=${encodeURIComponent(query)}`;
```

Comment:
Optionally, maybe make url query a bit easier to read. Not sure if is over-engineering though.

```
const urlQuery = [
      `lang=${lang.lc}`,
      `cc=${lang.cc}`,
      `query=${encodeURIComponent(query)}`,
    ].join('&');
const url = `${_endpoint}?${urlQuery}`;
```

Reply:
No all good, I like that pattern. I made the adjustment in https://github.com/carbon-design-system/ibm-dotcom-library/pull/696/commits/93ab1a74917d702fc7eefd7c6d569fe9c07c17d5.